verilog/analysis/checkers/enum_name_style_rule.cc

```diff
@@ -49,10 +50,19 @@ const char EnumNameStyleRule::kMessage[] =
 
 std::string EnumNameStyleRule::GetDescription(
     DescriptionType description_type) {
-  return absl::StrCat("Checks that ", Codify("enum", description_type),
-                      " names use lower_snake_case naming convention"
-                      " and end with '_t' or '_e'. See ",
-                      GetStyleGuideCitation(kTopic), ".");
+  static std::string basic_desc =
+      absl::StrCat("Checks that ", Codify("enum", description_type),
+                   " names use lower_snake_case naming convention"
+                   " and end with '_t' or '_e'. See ",
```

Comment:
.. `and also the additional optional regular expression.`

Otherwise the explanation looks a bit confusing to the user: tests for_t and _e ... and then there is a parameter with regular expressions - it is unclear from seeing that if that is replacing the _t, _e rule or augmenting it.

Similar in the other rules you added regexp: we should update the documentation to include the optional regexp capability.

Reply:
Ok so I changed the description as suggested, but I change the phrase to: `or also the additional optional regular expression.`, cause we can use either regex or the original rule, but when regex is not enabled we do not validate the old rules.